### DISCUSSION TITLE
Webhook and Action Automations do not start

### DISCUSSION THREAD

**Discussion by bart2801** [Budibase/budibase#12776] [Help] [answered]
Action Automations and Webhook Automations do not start when being triggered. The app is published, so that cannot be the problem. Other types of automations like "Row Updated" do start when triggered. I am on the Free pricing plan and have not yet reached the 200 automations limit. When I check the "Require confirmation" checkbox for the "Trigger Automation" action, the confirmation popup is shown when the action is triggered, but the automation does not start when I click Confirm. When I enter the webhook url of the Webhook Automation in the browser, the automation does not start and I am redirected to the app. The Automations History remains empty for the not started automations.

Is this a bug or am I doing something wrong?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @bart2801,

Can you provide some screenshots of the automations. Make sure you are using the bottom link for the webhook automation when sending data to it. The top link generates a schema.

The app action automation might not be working due to this issue [here](https://github.com/Budibase/budibase/issues/11287). You need to make sure a field has been created and that a value has been assigned to that field. (even if you aren't using the value of the field created.) If you don't have a value/field added the automation won't run. Try it with this change, hopefully this runs.

Thanks in advance.

**Reply by bart2801** [reply]
Hi @ConorWebb96,

You were right about the [issue](https://github.com/Budibase/budibase/issues/11287) which caused the problem. The issue is dated six months ago and is labeled 'medium priority'. For most people new to Budibase, this will be a blocking issue, so I think this should be 'high priority'. I already moved away from Budibase because of this, and I found UI Bakery which offers similar functionality.

My second issue was the Budibase webhook not functioning. It appears the webhook only supports POST requests. A GET request does not start the webhook automation.

Thanks for your support.
